src/ui_utils.c

```diff
@@ -2097,7 +2097,9 @@ static void on_config_file_clicked(GtkWidget *widget, gpointer user_data)
 		if (g_file_test(global_file, G_FILE_TEST_EXISTS))
 			g_file_get_contents(global_file, &global_content, NULL, NULL);
 
-		document_new_file(utf8_filename, ft, global_content);
+		// open or create the document and mark it as changed if it didn't already exist
+		GeanyDocument *doc = document_new_file(utf8_filename, ft, global_content);
+		document_set_text_changed(doc, ! g_file_test(file_name, G_FILE_TEST_EXISTS));
```

Comment:
> For the CSS file, it doesn't matter if it's just a blank file, it gets applied after the system one(s) and if it's blank it will do nothing. […]

The thing is that by default Geany fills up the file with the system version's content.  So if you save that, it'll overwrite the system forever.
Actually we probably shouldn't suggest to overwrite everything, although it's a fairly good thing to give an example of what the file should contain…